### ISSUE TITLE
2D Dodge The Creeps Tutorial - Missing/Deprecated UI elements in 4.0

### ISSUE THREAD

**Issue by TheYellowArchitect** [godotengine/godot-docs#6237]
**Your Godot version:**
4.0 beta1

**Issue description:**
https://docs.godotengine.org/en/latest/getting_started/first_2d_game/06.heads_up_display.html#scorelabel

In the above link, `Autowrap` does not have `On` choice, and I cannot find the `layout` property (I am a newbie)

Luckily, these 2 do not matter, as the images are enough for the player to make something approximate instead of 1:1 replica (it's just UI placement lol), but regardless, they should be accurate for 4.0

tl;dr: What is the proper `Autowrap` choice for the above, and where is the `layout` property?
